components-api/src/test/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidatorSpec.scala

```diff
@@ -13,10 +17,11 @@ class ParameterValidatorSpec extends AnyFunSuite with TableDrivenPropertyChecks
     forAll(
       Table(
         ("inputExpression", "isValid"),
-        ("''", false),
-        (" '' ", false),
-        ("\"\"", false),
-        (" \"\" ", false),
+        ("", false),
```

Comment:
Maybe instead of changing the test input, we should change the scope of the test? I mean to type the expressions and after that run the `isValid` method on the result. It will make these tests more "functional" then "unit" which is IMO a good approach.